README.md

```diff
@@ -8,15 +8,15 @@ This includes JBCRA frontend, serverless backend with TypeORM connected to auror
 
 ## Prerequisites
 
-NPM v7 (not officially released yet)
+NPM v7
 
-- `npm i -g npm@next-7`
+- `npm i -g npm@7`
 
 ## Quickstart
 
 ```shell
-npm i
-npm start
+git clone --depth=1 --branch=develop git@github.com:jetbridge/jkv2-ts-template.git myapp
+npm run template:init
```

Comment:
We should try this button https://github.com/jetbridge/jkv2-ts-template
<img width="450" alt="Screen Shot 2020-12-08 at 12 17 23 AM" src="https://user-images.githubusercontent.com/245131/101412143-c0052f00-38ea-11eb-85f4-efc8f5c6f029.png">